content/influxdb/cloud/reference/api/influxdb-1x/_index.md

```diff
@@ -14,4 +14,42 @@ related:
   - /influxdb/cloud/query-data/influxql
 ---
 
-{{< duplicate-oss >}}
\ No newline at end of file
+The InfluxDB v2 API includes InfluxDB 1.x compatibility endpoints that work with
+InfluxDB 1.x client libraries and third-party integrations like [Grafana](https://grafana.com) and others.
+
+<a class="btn" href="/influxdb/cloud/api/v1-compatibility/">View full v1 compatibility API documentation</a>
+
+## Authentication
+InfluxDB Cloud requires all query and write requests to be authenticated using
```

Comment:
what does this have to do with the compatibility endpoints? can we just say "In InfluxDB Cloud, when using the /query or /write apis, you will use your cloud username  as the user and a generated token as the password?

Reply:
https://docs.influxdata.com/influxdb/cloud/reference/api/influxdb-1x/#1x-compatible-authorizations